repos/spack_repo/builtin/packages/fdb/package.py

```diff
@@ -31,7 +31,7 @@ class Fdb(CMakePackage):
         values=any_combination_of(
             # FDB backend in indexed filesystem with table-of-contents with
             # additional support for Lustre filesystem stripping control:
-            "lustre",
+            "lustre"
```

Comment:
The comma here is preferred... otherwise it's going to be string concatenation when a line below the next comment is uncommented.

Reply:
if I set `skip-magic-trailing-comma = false`, `ruff` doesn't change this and `black` also doesn't complain. So I can preserve it -- not sure it's what we want b/c now the setting is different between the two.